rpc/src/client.rs

```diff
@@ -118,6 +118,11 @@ pub trait Client {
         self.perform(commit::Request::new(height.into())).await
     }
 
+    /// `/consensus_state`: get current consensus state (UNSTABLE)
```

Comment:
What is unstable here? The endpoint definition?

Reply:
I don't know exactly what's unstable about this - that's just what it says in the Go code for this endpoint 😁

Reply:
LOL this is how I used to mark APIs in Tendermint Go that weren't going to be covered by SemVer because they might change ... not sure it should be carried forward

Reply:
Cool, so I'll take out the "unstable" notice here 😁